### PR TITLE
Add Sidekiq Dashboard, update readme

### PR DESCRIPTION
**Pull request by ashkan18**
# Changes
 - Added Sidekiq Dashboard currently limited to admins and under `/admin/sidekiq`.
- Added `ArtsyAuthHelper` class which has common code needed to decode/verify a JWT and do some validation
- Refactored `AuthHandler` to use ☝️ 
- Updated README to include sidekiq dashboard urls and also removed query section from README since it gets out of date often and changed it to point to our Insomnia file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,8 +5,8 @@ Exchange aspires to be responsible for the various types of e-commerce interacti
 ## Meta
 
 * State: development
-* Production: https://exchange.artsy.net, [Admin Dashboard](https://exchange.artsy.net/admin)
-* Staging: https://exchange-staging.artsy.net, [Admin Dashboard](https://exchange-staging.artsy.net/admin)
+* Production: https://exchange.artsy.net, [Admin Dashboard](https://exchange.artsy.net/admin) | [Sidekiq Dashboard](https://exchange.artsy.net/admin/sidekiq)
+* Staging: https://exchange-staging.artsy.net, [Admin Dashboard](https://exchange-staging.artsy.net/admin) | [Sidekiq Dashboard](https://exchange-staging.artsy.net/admin/sidekiq)
```
Comment (williardx) [inline README.md]: This URL is 404'ing

Comment (ashkan18) [reply README.md]: yes, it will be available once this PR is merged (only to admins though)

File: lib/artsy_auth_helper.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+class ArtsyAuthHelper
```
Comment (williardx) [inline lib/artsy_auth_helper.rb]: What if this were called something like `ArtsyJWT` instead? `ArtsyAuthHelper` feels a little bit vague given that this class seems to only deal only with the JWT

Comment (ashkan18) [reply lib/artsy_auth_helper.rb]: interesting, my only issue with calling it `ArtsyJWT` is we include implementation detail in the name itself, lets say if in the future we changed it to KWT we don't want to change the class name.

Comment (williardx) [reply lib/artsy_auth_helper.rb]: Maybe `ArtsyToken` then?

Comment (ashkan18) [reply lib/artsy_auth_helper.rb]: yeah! makes sense 👍

**Comment by ashkan18** [comment]
merging 🙌

**Pull request merged**